pkg/metrics/metrics.go

```diff
@@ -16,6 +16,9 @@ var (
 	// UserSignupApprovedTotal is incremented each time a user signup is approved, can be multiple times per user if they reactivate multiple times
 	UserSignupApprovedTotal prometheus.Counter
 
+	// UserSignupApprovedWithMethodTotal is incremented each time a user signup is approved and includes either 'automatic' or 'manual' labels, can be multiple times per user if they reactivate multiple times
+	UserSignupApprovedWithMethodTotal *prometheus.CounterVec
```

Comment:
Just curios why you are using a pointer here while we don't use pointers in other counters? Just curios if there is any specific reason for this inconsistency.

Reply:
`prometheus.NewCounter()` returns `prometheus.Counter` but `prometheus.NewCounterVec()` returns `*prometheus.CounterVec` and I was trying to be consistent with *prometheus.GaugeVec that we have in the same file.